JsonFormReadme.md

```diff
@@ -0,0 +1,61 @@
+
+-----------------------------------------------------------------------------------------------
+# documentation   
+
+## required attributes
+
+1) key - a key is always string and enclosed in quotation mark
+
+2) name - specify the name of a feild
+
+3) label - used to create caption for a form control
+
+4) id - specify the unique id for an element
+
+5) type - defines the feild
+
+6) placeholder - shows information belongs in a feild
+
+7) validation - this value is a function that return a string errort or null if the feild is valid
+
+8) validation_server - it needs to validate on the server in addtion to on the client
+
+9) class - 
+
+10) wrapper class - Its provide all the of the componnets attribute and properties as props on
+
+the react component
+ 
+
+## validation
+
+1) required - this validates that the field has been provided and cannot be empty.
+
+2) min - this is the constraint to be put on the value and represents minimum acceptable value. 
+
+3) max - this is the constraint to be put on the value and represents maximum acceptable value. 
+
+4) Item - defines what appears in the array 
+
+5) optional - property should be considered optional 
+
+
+
+## field type 
+
+1) text - it is allow to enter basic values
+
+2) email - it a validation to only accept email addresses
+
+3) password - it doesn't display what the user types
+
+4) button - it requires a user tp perform an action
+
+5) checkbox - it can provide a list of choices 
```

Comment:
@sajana1829 No, a checkbox provides only one choice, which you can tick on or off. If you want a list you have to generate one checkbox for each list item.

Reply:
it allows us to select multiple option from  a list. i meant that.

Reply:
@sajana1829 The person reading this text will not know what you meant, they will only know what you have written. So write it clearly.

Reply:
ok , sure